src/components/OSCALLoader.js

```diff
@@ -132,8 +133,8 @@ export default function OSCALLoader(props) {
     setOscalUrl(event.target.value);
   };
 
-  const handleUuidChange = (event) => {
-    const newOscalUrl = `${props.backendUrl}/${props.oscalObjectType.restPath}/${event.target.value}`;
+  const handleUuidChange = (objectUuid) => {
+    const newOscalUrl = `${props.backendUrl}/${props.oscalObjectType.restPath}/${objectUuid}`;
```

Comment:
This breaks the API expected by the `OSCALLoaderForm` props correct? See Line 218?

Reply:
I may misunderstand what you mean, but `OSCALLoaderForm` was changed to deal with this.

https://github.com/EasyDynamics/oscal-react-library/blob/eb50a0174471509cd83c1068b8a7d0f12f5b3a42/src/components/OSCALLoaderForm.js#L113

`event.target.value` is the `uuid` here.